### ISSUE TITLE
Optionally grab OpenShift Crane access logs

### ISSUE THREAD

**Issue by aweiteka** [aweiteka/raas#27]
Story format...

> As a user I would like to process apache logs to gather
> metrics data on how often my data is being accessed.

Maybe add option to `status`?

**Comment by pschiffe** [comment]
This looks complicated - logs are stored in  app-root/logs/python.log file in openshift gear, but it seems that only way to get that file is only via ssh. And for that, ssh keys are needed... Is there some other way?

**Comment by aweiteka** [comment]
@pschiffe yes, this is complex. I believe we get the keys during upload but those would need to be stored in the configuration. We should make sure we have the complete story for this (what will parse these logs? etc) before doing this work. Maybe a "companion" OpenShift app could be used to perform this work? Maybe some partners want to integrate with an existing system.

**Comment by pschiffe** [comment]
OK, let's wait for the complete user story and propose appropriate solution afterwards.
